.ci/jenkins/Jenkinsfile.promote

```diff
@@ -557,8 +557,8 @@ void runMavenDeploy(MavenCommand mvnCmd) {
 void removeJbossNexusFromMavenAndGradle() {
     sh "sed -i \':a;N;\$!ba;s/\\n *<repositories>.*<\\/repositories>//g\' */pom.xml"
     sh "sed -i \':a;N;\$!ba;s/\\n *<repositories>.*<\\/repositories>//g\' pom.xml"
```

Comment:
With the change of the `optaplanner-quickstarts` structure, I think also these two lines need to be replaced by something similar:
```
find . -name pom.xml -print0 | xargs -0 sed -i \':a;N;\$!ba;s/\\n *<repositories>.*<\\/repositories>//g\'
```

Reply:
updated